### DISCUSSION TITLE
SQLAlchemy 1.4 support

### DISCUSSION THREAD

**Discussion by kraanerk** [apache/superset#18078] [Ideas]
What about upgrading SQLAlchemy from 1.3 to 1.4?

**Comment by villebro** [comment]
While upgrading SQLAlchemy shouldn't be too difficult, I'm unsure if all the SQLAlchemy connectors out there are quite yet ready for 1.4. In addition, we currently have multiple dependencies that either don't work or may not work correctly with 1.4 (e.g. `Flask-Sqlalchemy`). But if you are interested in opening a draft PR that upgrades Superset to support SQLAlchemy 1.4, I'd be happy to help with reviewing/testing. The draft PR could be kept open and rebased until all blockers are addressed.

**Comment by msardana94** [comment]
[snowflake-sqlalchemy](https://pypi.org/project/snowflake-sqlalchemy) has bumped the sqlalchemy version to be >=1.4 which causes package conflicts with superset unless we use `snowflake-sqlalchemy` package version <1.3.
